src/components/Layout/index.jsx

```diff
@@ -102,12 +102,16 @@ export default class Layout extends React.Component {
           { [styles.hasBack]: Boolean(backBtn) }
         )}
       >
-        {isNormal && !isHome && <TitleBanner title={title} hasSearch={hasSearch} />}
         {hasMenu && <SideNav isScroll={isScroll} />}
         {noNotification ? null : <Notification />}
-        {backBtn}
         <Loading isLoading={isLoading} className={styles[loadClass]}>
-          {children}
+          {!isLoading &&
+            <Fragment>
+              {isNormal && !isHome && <TitleBanner title={title} hasSearch={hasSearch} />}
```

Comment:
`children` in Loading component is lazy loaded content, since `TitleBanner`, `backBtn` is no need to lazy load, so your refactor is no need.

Reply:
In Loading Component, last line of code: return <Fragment>{children}</Fragment>, page rendering before  componentDidMount. TitleBanner, backBtn  mybe no need to judge isLoading.